templates/item-info.widget

```diff
@@ -152,64 +152,84 @@ function editItemInfo(itemUid) {
   switchSection("#item-" + itemUid + " .item-info", "editing");
 }
 
+CSS
+------------------------------------------------------------
+.formLabel {
+  display: block;
+  margin-bottom: 5px;
+  margin-top: 15px;
+}
+
+.form-group {
+  margin-top: 15px;
+  margin-bottom: 5px;
+}
+
+.form-btn-group {
+  margin-top: 20px;
+  margin-bottom: 5px;
+}
+
+.save {
```

Comment:
I think you should rename this class because it will clash with classes elsewhere.

(All CSS from templates is concatenated into one file.)